net.sourceforge.qtpfsgui.LuminanceHDR.svg

```diff
@@ -0,0 +1,528 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This should be kept in code repository so all distributions can benefit.

Reply:
This will be since the next application release.